loops-stevie-athen.js

```diff
@@ -0,0 +1,61 @@
+// 1. Logging values with for loops
+// Write a for loop that logs each number from 1-20
+
+for(let i=1; i<21; i++){
+  console.log(i)
+}
+// Write a for loop that logs the result of each number from 1-20 tripled.
+for(let i=1; i<21; i++){
+    console.log(i * 3)
+}
+// Create a for loop that logs each even number from 1-20, and in the place of every odd number, returns the word "ODD"
+for(let i=1; i<21; i++){
+    if(i % 2 !== 0){
+        console.log("ODD")
+    }else{
+        console.log(i)
+    }
+
+}
+// 2. Looping over an array. Consider this variable:
+// Create a loop that will log the highest number from the array. Expected output --> 67
+
+var nums = [3, 57, -9, 20, 67]
+
+for(let i=0; i<1; i++){
+    let orderedNums = nums.sort((a,b)=>a-b)
+    let highestNumber = nums.length - 1
+    console.log(orderedNums[highestNumber])
+}
+
+// Create a loop that will log the lowest number from the array
+for(let i=0; i<1; i++){
+  let orderedNums = nums.sort((a,b)=>a-b)
+  let lowestNumber = nums.length - nums.length
+  console.log(orderedNums[lowestNumber])
+}
+// Create a loop that will log the remainder of each number when divided by 2. Expected output --> 1, 1, -1, 0, 1
```

Comment:
The expected output did not display because the array has been altered by the .sort() mutator method. If you console.log(nums), the results will be `[ -9, 3, 20, 57, 67 ]`. Therefore the results are 
-1
1
0
1
1